app/controllers/tag_lists_controller.rb

```diff
@@ -0,0 +1,51 @@
+class TagListsController < ListController
+  self.nesting = Group
+
+  skip_authorization_check
+  skip_authorize_resource
+
+  helper_method :group
+
+  respond_to :js, only: [:new, :deletable]
+
+  def create
+    new_tags = tag_list.build_new_tags
```

Comment:
Schöner fände ich, wenn mehr an das domain objekt delegiert werden könnte, zb. 

`Tag::List.new(people, tags).add` bzw `Tag::List.new(people, tags).remove`

Das Filtern der Personen Liste würde ich in diesem Fall im Controller machen, dann müssten wir die Ability auch nicht weiterreichen und könnten trotzdem via flash[:alert] informieren, dass auf bestimmten Personen keinen Rechte vorhanden sind